meson.build

```diff
@@ -34,7 +34,7 @@ endif
 #   2. Backward-compatible ABI change: bump minor, reset patch
 #   3. Other, eg. bugfix: bump patch
 abi_version_major = 0
```

Comment:
We should probably bump this to 1 to distinguish from 0.x binaries with incompatible ABIs.

Reply:
ooop! good point

Reply:
Ooop! Missed that, thanks!